rpms/postgres-decoderbufs.spec

```diff
@@ -0,0 +1,47 @@
+Name:		postgres-decoderbufs
+Version:	0.9.5
+Release:	1%{?dist}
+Summary:	PostgreSQL Protocol Buffers logical decoder plugin
+
+License:	MIT
+URL:		https://github.com/debezium/postgres-decoderbufs
+
+%global full_version %{version}.Final
+
+Source0:	https://github.com/debezium/%{name}/archive/v%{full_version}.tar.gz
+
+BuildRequires:	gcc
+BuildRequires:	postgresql-devel >= 9.6, postgresql-server-devel >= 9.6
```

Comment:
So on that one: we can have a single RPM which works with all PG versions starting with 9.6?

Reply:
@gunnarmorling Exactly

Reply:
Ok.